libr/core/cmd_type.c

```diff
@@ -267,8 +267,22 @@ static void save_parsed_type(RCore *core, const char *parsed) {
 	}
 }
 
+static RCore *core_ = NULL; // HACK
+
 static int stdifstruct(void *user, const char *k, const char *v) {
-	return !strncmp (v, "struct", strlen ("struct") + 1);
+	if (!strncmp (v, "struct", strlen ("struct") + 1)) {
+		return true;
+	} else if (!strcmp (v, "typedef")) {
```

Comment:
no need to else after return

Reply:
done 2a9a424fa6527c0b43a725c75f6b26bd0be3f848